staging/src/k8s.io/client-go/examples/third-party-resources/main.go

```diff
@@ -1,170 +0,0 @@
-/*
```

Comment:
> why are examples removed?

Just ran a sync.  I guess the script isn't right anymore.  Miss an update maybe?